annotations/src/main/resources/org/leandreck/endpoints/templates/typescript/service.ftl

```diff
@@ -161,6 +175,19 @@ export class ${serviceName} {
 
 <#--</#list>-->
 
+    private getHttpParams(data: any): HttpParams {
+        const params: HttpParams = new HttpParams();
+
+        Object.keys(data).forEach((key: string) => {
+            const value: any = data[key];
+            if (value != null) { // Check for null AND undefined
+                params.set(key, String(value));
```

Comment:
technically this should not work as HttpParams is immutable and the returned new instance is not assigned